recipes/wasmtime-cpp/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+import os
+import shutil
+
+required_conan_version = ">=1.33.0"
+
+class WasmtimeCppConan(ConanFile):
+    name = 'wasmtime-cpp'
+    homepage = 'https://github.com/bytecodealliance/wasmtime-cpp'
+    license = 'Apache-2.0'
+    url = 'https://github.com/conan-io/conan-center-index'
+    description = "Standalone JIT-style runtime for WebAssembly, using Cranelift"
+    topics = ("webassembly", "wasm", "wasi", "c++")
+    settings = "compiler"
+    exports_sources = "include/*"
+    no_copy_source = True
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 17
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "16",
+            "apple-clang": "9.4",
+            "clang": "5.0",
+            "gcc": "8.0"
+        }
+
+    def requirements(self):
+        self.requires("wasmtime/0.29.0")
+
+    def validate(self):
+        compiler = self.settings.compiler
```

Comment:
```suggestion
        if self.settings.compiler.get_safe("cppstd"):
            tools.check_min_cppstd(self, 20)
        compiler = self.settings.compiler
```

Reply:
tools.check_min_cppstd(self, 20) it is wrong !!
This library supports C++ beginning from C++17

Reply:
I thought it needed c++20 because it used `std::span`: https://github.com/bytecodealliance/wasmtime-cpp/issues/1#issuecomment-867186649
But it looks like they use library feature test-macros: https://github.com/bytecodealliance/wasmtime-cpp/blob/40fc6f8dea7e90e5d56d51d89a9be8922b0113ee/include/wasmtime.hh#L51

Reply:
Yes, it was added by me )